server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -295,8 +306,17 @@ func (e *EventChannel) handleEvent(event *pepb.PublishBuildToolEventStreamReques
 		if err := e.env.GetInvocationDB().InsertOrUpdateInvocation(e.ctx, ti); err != nil {
 			return err
 		}
+	} else if !e.hasReceivedStartedEvent {
+		e.eventsBeforeStarted = append(e.eventsBeforeStarted, event)
+		if len(e.eventsBeforeStarted) > 100 {
```

Comment:
can you make 100 a const and change it to 10?

Reply:
Done